x-pack/plugins/alerts/server/alert_type_registry.ts

```diff
@@ -108,6 +123,13 @@ export class AlertTypeRegistry {
           this.taskRunnerFactory.create(normalizedAlertType, context),
       },
     });
+    // No need to notify usage on basic alert types
+    if (alertType.minimumLicenseRequired !== 'basic') {
```

Comment:
nit.
Is there an enum we can use to reference the `basic` string?
I'm always worried when we hard code values like this.